Makefile

```diff
@@ -44,6 +46,9 @@ start-dockerdemo:
 clean-dockerdemo:
 	scripts/dockerhost/cleanup-dockerhosts
 
+ssh:
```

Comment:
`make demo` always exits 0. the problem is that make demo calls build, which calls several other tasks. I'd like to avoid having those run just to ssh in.

Would this work?

`vagrant ssh netplugin-node1 || echo 'Please run "make demo"'`